doc/content/source/problems/NekRSSeparateDomainProblem.md

```diff
@@ -37,67 +55,86 @@ Pressure information is transferred between NekRS and the 1D T/H code using a gl
 
 ## Transferring data between NekRS and the 1D T/H code
 
-A summary of postProcessors generated is shown in the following table. The creation of these postProcessors depends on (1) the `coupling_type` given and (2) whether or not NekRS is solving for temperature.
+A summary of postProcessors generated is shown in the following table for velocity, temperature, and scalar01 coupling.
+ PostProcessors for scalar02 and scalar03 coupling follow the same pattern as scalar01.
+The postProcessor created depends on (1) the `coupling_type` given, (2) whether or not NekRS is solving for temperature
+ and (3) the optional `coupling_scalars` provided.
 
-| postProcessor created | inlet | outlet | inlet outlet |
+| postProcessor created | `coupling_type = inlet` | `coupling_type = outlet` | `coupling_type = inlet outlet` |
 | - | - | - | - |
 | dP | $\checkmark$ | $\checkmark$ | $\checkmark$ |
 | inlet_V | $\checkmark$ | &nbsp; | $\checkmark$ |
 | outlet_V | &nbsp; | $\checkmark$ | $\checkmark$ |
 | inlet_T | $\checkmark$, if temperature solved | &nbsp; | $\checkmark$, if temperature solved |
 | outlet_T | &nbsp; | $\checkmark$, if temperature solved | $\checkmark$, if temperature solved |
+| inlet_S01 | $\checkmark$, if scalar01 in `coupled_scalars` | &nbsp; | $\checkmark$, if scalar01 in `coupled_scalars` |
+| outlet_S01 | &nbsp; | $\checkmark$, if scalar01 in `coupled_scalars` | $\checkmark$, if scalar01 in `coupled_scalars` |
+
 
 
-Example MultiApp transfers are shown below for `coupling_type = 'inlet outlet'` with transfers between the NekRS MainApp and the `1D_TH_code` SubApp. These transfers include all possible postProcessors created.
+Example MultiApp transfers for velocity, temperature and scalar01 coupling are shown below for
+ `coupling_type = 'inlet outlet'` and `coupled_scalars = 'scalar01'`. These transfers assume NekRS
+ is acting as the MainApp and the `1D_TH_code` as the SubApp.
```

Comment:
I would add another sentence making it clear that Nek doesn't *need* to be the main app, but just that the documentation shows this example.